prettier.config.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
-    trailingComma: "all",
+    trailingComma: 'all',
```

Comment:
if you want to keep this file you will have to require it at the top of eslintrc.js and then use the config in place of the prettier config in there already. Alternatively you can just copy this config into eslintrc and delte this

Reply:
Not sure I understand here, my file is being respected with out any import.  If I remove the file it forces a load of format changes I don't want.

Reply:
Aligned prettier to the root config.